README.md

```diff
@@ -26,10 +26,13 @@ Refer to the README files in each subdirectory of this repo for specific instruc
 * [TypeScript](http://www.typescriptlang.org/)
 
 ## Contributing
-Anyone and everyone is welcome to contribute. We accept pull requests. Feel free to:  
-* Report issues
-* Contribute code
-* Improve documentation
+
+Esri welcomes contributions from anyone and everyone. Please see our [guidelines for contributing](https://github.com/esri/contributing).
+
+We use issues and pull requests to address problems found in the [ArcGIS API for Javascript](http://js.arcgis.com/) here.  If you're asking for help please utilize the supplementary resources below.
```

Comment:
what about #33? 

either way, i think it'd be fine to just ditch that sentence.

> ~~Issues and pull requests to address problems found in the [ArcGIS API for Javascript](http://js.arcgis.com/) here.~~ If you're asking for help please utilize the supplementary resources below.

i hadn't thought of the fact that users are directed to CONTRIBUTING.md doc when they open issues and pull requests. thats a good point.